lib/loadMochaOptions.js

```diff
@@ -0,0 +1,58 @@
+const { Command } = require("commander");
+
+/**
+ * Parse list.
+ */
+const list = str => str.split(/ *, */);
+
+/**
+ * Parse multiple flag.
+ */
+const collect = (val, memo) => memo.concat(val);
+
+module.exports = function loadMochaOptions(args) {
```

Comment:
@alexjeffburke This is also lifted out of `bin/_mocha`. I think the most elegant way is to duplicate the necessary bits at first. If we are hugely successful, we might be able to convince mocha-core to pull it out into requireable methods.